api/src/test/java/org/pmiops/workbench/institution/InstitutionServiceTest.java

```diff
@@ -121,51 +136,40 @@ public void test_getInstitutions() {
   @Test
   public void test_getInstitutionsWithInstruction() {
     final String instructions = "Do some magic!";
-    final InstitutionUserInstructions inst =
-        new InstitutionUserInstructions()
-            .institutionShortName(roundTrippedTestInst.getShortName())
-            .instructions(instructions);
-    service.setInstitutionUserInstructions(inst);
+    roundTrippedTestInst.setUserInstructions(instructions);
+    service.updateInstitution(roundTrippedTestInst.getShortName(), roundTrippedTestInst);
     List<Institution> institutionList = service.getInstitutions();
     assertThat(institutionList.get(0).getUserInstructions()).contains(instructions);
   }
 
   @Test
   public void test_getInstitutionsWithoutInstruction() {
     List<Institution> institutionList = service.getInstitutions();
-    assertThat(institutionList.get(0).getUserInstructions()).isNull();
+    assertThat(institutionList.get(0).getUserInstructions()).isEmpty();
   }
 
   @Test
   public void test_getInstitution() {
+    roundTrippedTestInst.setShortName(testInst.getShortName());
     assertThat(service.getInstitution(testInst.getShortName())).hasValue(roundTrippedTestInst);
     assertThat(service.getInstitution("otherInst")).isEmpty();
 
     final Institution otherInst =
         new Institution()
-            .shortName("otherInst")
             .displayName("The Institution of testing")
+            .duaTypeEnum(DuaType.MASTER)
+            .organizationTypeEnum(OrganizationType.ACADEMIC_RESEARCH_INSTITUTION)
             .emailAddresses(Collections.emptyList())
-            .emailDomains(Collections.emptyList());
-    service.createInstitution(otherInst);
-    assertThat(service.getInstitution("otherInst")).hasValue(otherInst);
-  }
-
-  @Test
-  public void test_updateInstitution_shortName() {
```

Comment:
If we can't update the shortName anymore (I think that's a good idea) let's be explicit about it.  Keep this test, but show that it throws an exception.